src/main/webapp/tinyMce.jsp

```diff
@@ -10,81 +10,84 @@
 <%@ include file="functions.jsp" %>
 <%@ page import="de.uni_tuebingen.ub.nppm.util.*" isThreadSafe="false" %>
 
-<%    if (AuthHelper.isAdminLogin(request)) {
 
+
+<%
+    int benutzerID = ((Integer) session.getAttribute("BenutzerID")).intValue();
+    Benutzer benutzer = BenutzerDB.getById(benutzerID);
+    boolean isAdmin = benutzer.isAdmin();
```

Comment:
And same here again

Reply:
done look at commit: isAdminRequired function